src/test/scala/io/citrine/lolo/trees/splits/ClassificationSplitterTest.scala

```diff
@@ -0,0 +1,59 @@
+package io.citrine.lolo.trees.splits
+
+import io.citrine.lolo.TestUtils
+import io.citrine.lolo.encoders.CategoricalEncoder
+import io.citrine.theta.Stopwatch
+
+
+/**
+  * Hold some performance profiling routines
+  */
+class ClassificationSplitterTest {
```

Comment:
Is meant to evaluate the performance against some benchmark, or just print the time?